app/routes/events/view/scheduler.js

```diff
@@ -22,8 +22,42 @@ export default Route.extend({
       }
     ];
 
+    let scheduledFilterOptions = [
+      {
+        and: [
+          {
+            name : 'starts-at',
+            op   : 'ne',
+            val  : null
+          },
+          {
+            name : 'ends-at',
+            op   : 'ne',
+            val  : null
+          }
+        ]
+      }
+    ];
+
     let eventDetails = this.modelFor('events.view');
 
+    let scheduledSessions = await eventDetails.query('sessions', {
+      include : 'speakers',
+      filter  : scheduledFilterOptions
+    });
+
+    let scheduled = []; // to convert sessions data to fullcalendar's requirements
+    scheduledSessions.forEach(function(session) {
+      session.get('microlocation').then(microlocation => {
```

Comment:
- The formatting is not consistent with rest of the app. Put `.then` in the next line.
- Add a `.catch()` clause to handle failure while fetching microlocation.